src/Aspire.Hosting.Azure.SignalR/SignalREmulatorContainerImageTags.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Azure.SignalR;
+internal static class SignalREmulatorContainerImageTags
+{
+    /// <summary>mcr.microsoft.com</summary>
+    public const string Registry = "mcr.microsoft.com";
+
+    /// <summary>signalr/signalr-emulator</summary>
+    public const string Image = "signalr/signalr-emulator";
+
+    /// <summary>latest</summary>
+    public const string Tag = "latest"; // latest is the only arch-agnostic tag
```

Comment:
is the comment true or was it copied from 

https://github.com/dotnet/aspire/blob/29a23f3c5eb089c30fc11fcc0b411244b2708a0a/src/Aspire.Hosting.Azure.EventHubs/EventHubsEmulatorContainerImageTags.cs#L15

Reply:
Sorry I forgot to delete it, let me remove it